app/models/email_log.rb

```diff
@@ -85,6 +87,14 @@ def bounce_key
     super&.delete('-')
   end
 
+  def cc_users
+    return [] if !self.cc_user_ids
+    @cc_users ||= User.where(id: self.cc_user_ids)
+  end
+
+  def cc_addresses_split
+    @cc_addresses_split ||= self.cc_addresses&.split(";") || []
+  end
```

Comment:
Will this methods be used in a future PR? Seems odd to me that we're adding methods which are only used in tests.

Reply:
Yes they will be used later.